tests/src/ondevice_tests/pipeline/node/camera_fps_config_test_part_2.cpp

```
@@ -0,0 +1,129 @@
+        std::cout << "Testing num frames = " << overrideQueueSize << "\n" << std::flush;
+        dai::Pipeline pipeline;
+        std::map<dai::CameraBoardSocket, std::vector<std::tuple<int, int, float>>> streamsRvc4{
+            // Has to be (for now):
+            // - without FpsRegulator (different fps per same sensor)(different fps on different sensors also doesn't work right now)
+            // - without ManipResizer so size should be supported by ISP directly
+            {dai::CameraBoardSocket::CAM_A, {{640, 480, 30.0f}, {1920, 1440, 30.0f}}},
+            {dai::CameraBoardSocket::CAM_B, {{640, 400, 30.0f}, {1280, 800, 30.0f}}},
+            {dai::CameraBoardSocket::CAM_C, {{640, 400, 30.0f}, {1280, 800, 30.0f}}},
+        };
+        // RVC2 is more RAM bound so use smaller sizes for the test
+        std::map<dai::CameraBoardSocket, std::vector<std::tuple<int, int, float>>> streamsRvc2{
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)